app/views/idv/gpo/index.html.erb

```diff
@@ -12,8 +12,8 @@
 <%= render PageHeadingComponent.new.with_content(@presenter.title) %>
 
 <p>
-  <%= t('idv.messages.gpo.address_on_file') %>
-  <%= t('idv.messages.gpo.timeframe') %>
+  <%= sanitize t('idv.messages.gpo.address_on_file') %>
```

Comment:
I think we tend to prefer approach where we rename the i18n keys to end in _html so that the i18n framework automatically marks them as `html_safe`

Reply:
Aha, I did not know about this capability. I've updated in https://github.com/18F/identity-idp/pull/6815/commits/bf565b174bffd6d8737c6775978ef2d003c9c410 Thanks!